man/man1/projinfo.1

```diff
@@ -89,7 +89,7 @@ The default is crs.
 .TP
 .B \-\-summary
 When listing coordinate operations available between 2 CRS, return the
-result in a summary format, mentionning only the name of the coordinate
+result in a summary format, mentioning only the name of the coordinate
```

Comment:
No need to make changes here. The man pages are auto generated from sphinx at release time.

Reply:
I figured so. Did you want this change backed out, or is it fine to leave in for now?

Reply:
If it's not too much trouble, yes. I don't like to introduce unnecessary clutter to the git history. At some point I will se if we can remove these files altogether and generate them as part of "make dist" when preparing releases.

Reply:
Done. And I've squashed these commits into one.